3p-ecosystem/build.gradle.kts

```diff
@@ -113,6 +113,10 @@ android {
     }
```

Comment:
Delete the above compileOptions and kotlinOptions blocks (and associated comment) - the jvmToolchain at the top takes care of these (and if jvmToolchain is changed later it would be easy to forget to change these)

Reply:
done